learning/tour-of-beam/backend/internal/entity.go

```diff
@@ -0,0 +1,69 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
```

Comment:
Are you sure that it's ok that the internal package associated with entities? Maybe should we create a model package or smth like that?

Reply:
It is very controversial subject where to keep models for a Go project
I usually stick to Clean Architecture approach, and they suggest placing models in the project root.
But that will create a dependency cycle for Cloud Function, so I decided to move it in the `internal` module